### DISCUSSION TITLE
how to set up param.json and machine.json of dpgen in local machine?

### DISCUSSION THREAD

**Discussion by McAlexDudu** [deepmodeling/dpgen#662] [Q&A] [answered]
Hi, it seems little guide about how to set up param.json and machine.json on local machine(centos7,2CPU,88cpus). I run it and got error like this:
![error](https://user-images.githubusercontent.com/16278257/157553198-a5e89ae5-0749-4a4e-bb84-6b0f1accc2fd.PNG)
[machine.txt](https://github.com/deepmodeling/dpgen/files/8218754/machine.txt)
[param.txt](https://github.com/deepmodeling/dpgen/files/8218755/param.txt)

**Comment by njzjz** [comment, marked as answer]
As it is indicated on the screen,
> Please check the dirs and scripts in remote root. The job information mentioned above may help

**Reply by McAlexDudu** [reply]
Should I write the validation systems of training in param.json? I saw some examples don't give it
<img width="1009" alt="截屏2022-03-10 08 36 20" src="https://user-images.githubusercontent.com/16278257/157567281-9bade4dd-c096-4997-b550-7792867c3a8f.png">
<img width="1009" alt="截屏2022-03-10 08 36 45" src="https://user-images.githubusercontent.com/16278257/157567288-fd531934-1372-4fd1-a8f0-e442569c2c34.png">
